python/taichi/lang/impl.py

```diff
@@ -301,6 +310,8 @@ def var(dt, shape=None, offset=None, needs_grad=False):
             "right after 'import taichi as ti' if you are using Jupyter notebook."
         )
 
+    del _taichi_skip_traceback
```

Comment:
Just curious, what happens if we don't delete this?

Reply:
If delete: exceptions from codes below this line will be included in traceback.
If not delete: exceptions from codes below this line will not be included in traceback.